frontend/src/pages/Search/Search.js

```diff
@@ -74,14 +86,37 @@ class Search extends Component {
         })
     };
 
+    queryRides = () => {
+        // this.props.dispatch({type: SEARCH_RIDES_SUCCESS});
+        fetch(xurl)
+            .then(res => res.json())
+            .then(res => {
+                console.log(res); /*****/
+                const queried_rides = [];
+                for (let ride in res) {
+                    queried_rides.push(
+                        [
+                            res[ride].startLoc.city + ', ' + res[ride].startLoc.state,
+                            res[ride].endLoc.city + ', ' + res[ride].endLoc.state,
+                            new Date(res[ride].time),
+                        ]
+                    )
+                }
+                this.setState({
+                    filteredRides: queried_rides
+                });
+                console.log(queried_rides);
+            })
+    };
+
     // helper function for filterRides()
     checkMatch = (dest, dateArray, date) => {
```

Comment:
we might not need this since the backend will filter the search and return the queried results